app/gui/src/project-view/components/GraphEditor.vue

```diff
@@ -108,15 +110,13 @@ const graphNavigator: GraphNavigator = provideGraphNavigator(viewportNode, keybo
 
 // === Client saved state ===
 
-const persisted = providePersisted(
+providePersisted(
```

Comment:
Looks like persisted store `doc`, `rtab` and `rwidth` properties are no longer being used, so the right dock size or documentation open state will no longer be persisted. Is this intentional?

Reply:
This is moved to app/gui/src/providers/rightPanel.ts#L157 I remove it from here.